.yarnrc.yml

```diff
@@ -1,3 +1,5 @@
+nodeLinker: node-modules
```

Comment:
This is to disable yarn's Plug-n-Play functionality cuz it's kinda glitchy and weird